src/EspressoNitroTEEVerifier.sol

```diff
@@ -33,20 +37,20 @@ contract EspressoNitroTEEVerifier is NitroValidator, IEspressoNitroTEEVerifier,
         Ownable()
     {
         _certManager = certManager;
-        registeredEnclaveHash[enclaveHash] = true;
+        registeredBatchPosterEnclaveHashes[enclaveHash] = true; // TODO: modify this constructor after some review on this.
         _transferOwnership(msg.sender);
     }
 
     /**
-     * @notice This function registers a new signer by verifying an attestation from the AWS Nitro Enclave (TEE)
+     * @notice This function registers a new Caff Node by verifying an attestation from the AWS Nitro Enclave (TEE)
      * @param attestation The attestation from the AWS Nitro Enclave (TEE)
      * @param signature The cryptographic signature over the COSESign1 payload (extracted from the attestation)
      */
-    function registerSigner(bytes calldata attestation, bytes calldata signature) external {
+    function registerCaffNode(bytes calldata attestation, bytes calldata signature) external {
         Ptrs memory ptrs = validateAttestation(attestation, signature);
         bytes32 pcr0Hash = attestation.keccak(ptrs.pcrs[0]);
-        if (!registeredEnclaveHash[pcr0Hash]) {
-            revert InvalidAWSEnclaveHash();
+        if (!registeredCaffNodeEnclaveHashes[pcr0Hash]) {
+            revert InvalidAWSEnclaveHash(pcr0Hash);
```

Comment:
you can include the service type in this error log e.g.
```suggestion
            revert InvalidAWSEnclaveHash(pcr0Hash, ServiceType.CaffNode);
```

also in this case, is it more accurate to say the enclaveHash is not valid or not registered?

Reply:
I have added this in a commit, but I am not sure I understand what you mean by saying it's not valid or not registered?